regtests/bin/matrix_ukmo_cray

```diff
@@ -141,6 +142,13 @@ fi
 # 2.  Execute matrix.base ...                                                 #
 # --------------------------------------------------------------------------- #
              
+# Compiler configuration, required for coupling
```

Comment:
@ukmo-juan-castillo 
Can you move these `cmplr` lines into the `$comp` conditional sections starting at line 63, please?
You can then delete line 87 that is actually set incorrectly (not by you).

Reply:
This change has been now addressed and has been pushed in the fb branch. Waiting for comments from other reviewers regarding the changes in matrix.base.